addons/hr_timesheet/static/src/components/project_task_progress/project_task_progress.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { IntegerField, integerField } from "@web/views/fields/integer/integer_field";
+
+export class ProjectTaskProgress extends IntegerField {
+    get formattedValue() {
+        return `(${super.formattedValue}%)`;
+    }
+}
+
+export const projectTaskProgress = {
+    ...integerField,
+    component: ProjectTaskProgress,
+};
+
+registry.category("fields").add("project_task_progress", projectTaskProgress);
```

Comment:
Why not using percentage widget instead of doing customisation? 🤔

Reply:
because the value is multiplied by 100 
https://github.com/odoo/odoo/blob/054ca0a19aaf297f420a1b478b93ae26f1b943b8/addons/web/static/src/views/fields/formatters.js#L296

Reply:
If it is just for that reason then why not changing the compute method and doing a migration script to divide by 100? 

I am not sure if it is a real issue to change the compute method. 🤔

Reply:
there is another reason that we also need to add parenthesis like i did in (${super.formattedValue}%),
changing the compute will impact some code (but nothing blocking as i can see)
e.g https://github.com/odoo/odoo/blob/d0c2e9344b6639e41ab6e274627ac650dcfd8361/addons/hr_timesheet/views/project_task_views.xml#L168-L169
we will need to compare with with 1 instead of 100 and it's okay

but to get the (), we will need to inherit the % field.
so actually: it's just a new field 
by inheriting % field: new field + changing on compute + migration + some adaptation.

i prefer we stay on the actual solution. what do you think ?